### PR TITLE
Handle sum of memory percentage when memory_limit is 0

### PR DESCRIPTION
**Pull request by jsignell**
Closes https://github.com/dask/dask/issues/6452

To test do: 

```python
from distributed import LocalCluster, Client

cluster = LocalCluster(n_workers=2, memory_limit=0)
client = Client(cluster)

client
```

and navigate to http://127.0.0.1:8787/workers

### CODE REVIEW

**Comment by quasiben** [comment]
Tested this out and it worked nicely.  Thanks @jsignell!
Screen Shot 2020-07-24 at 9 56 32 AM

**Comment by mrocklin** [comment]
Thanks @jsignell !

Do we want to add a small test?  There are a variety of WorkerTable tests in `distributed/dashboard/tests/test_scheduler_bokeh.py`

**Comment by jsignell** [comment]
Ah! Yes I always forget that these have tests :grimacing: I'll add one now.

**Comment by lesteve** [comment]
This does seems like a edge case I missed in #3897, thanks for the fix @jsignell!

File: distributed/dashboard/tests/test_scheduler_bokeh.py [hunk 1]
```diff
@@ -430,6 +430,19 @@ def metric(worker):
     assert s.workers[a.address].metrics["metric"] == -999
 
 
+@gen_cluster(client=True)
+async def test_WorkerTable_with_memory_limit_as_0(c, s, a, b):
+    s.workers[a.address].memory_limit = 0
+    s.workers[b.address].memory_limit = 0
```
Comment (mrocklin) [inline distributed/dashboard/tests/test_scheduler_bokeh.py:436]: I think that something like this should work instead, and probably relies only on public API rather than reaching in

```suggestion
@gen_cluster(client=True, worker_kwargs={"memory_limit": 0})
async def test_WorkerTable_with_memory_limit_as_0(c, s, a, b):
```

Comment (mrocklin) [reply distributed/dashboard/tests/test_scheduler_bokeh.py:436]: +1 from me otherwise

Comment (jsignell) [reply distributed/dashboard/tests/test_scheduler_bokeh.py]: Oh nice! I didn't look closely at what was going on in the test setup :facepalm:

**Comment by mrocklin** [comment]
Thanks @jsignell !

**Comment by rcthomas** [comment]
Thanks @jsignell !
